src/documentation/explore/ToolkitTopLevelListItem.tsx

```diff
@@ -41,9 +44,17 @@ const ToolkitTopLevelListItem = ({
     >
       <VStack alignItems="stretch" spacing={1} flex="1 1 auto">
         <HStack justifyContent="space-between">
-          <Text as="h3" fontSize="lg" fontWeight="semibold">
-            {name}
-          </Text>
+          <HStack>
+            <Text as="h3" fontSize="lg" fontWeight="semibold">
+              {name}
+            </Text>
+            {isV2Only && (
+              <Tag fontWeight="semibold" background="brand.500" color="gray.25">
+                V2
+              </Tag>
+            )}
+          </HStack>
```

Comment:
Maybe this should be a component?

Reply:
[`8cc14ff2ff`](https://github.com/microbit-foundation/python-editor-next/pull/443/commits/8cc14ff2fff445b903115f94c87ea845d7f71e49)